pip/dev_requirements.txt

```diff
@@ -0,0 +1,10 @@
+bokeh
+matplotlib
+matplotlib-venn
+seaborn>=0.13.0
+umap-learn
+bravado
+maestrowf
+MolVS
+mordred
+ipykernel
```

Comment:
I'm not familiar with all of these dependencies. Some may need to be moved back over to `cpu_requirements.txt`. I will double check this, but feel free to call out any mistakes in this separation